README.md

```diff
@@ -109,10 +127,11 @@ given in [document](https://github.com/shubhamagrawal6629/AlternateBitProtocolSi
 5. For windows system type
 >               ./SUBNET_TEST.exe
```

Comment:
For windows, it actually works without '.exe'.

Reply:
are you using cygwin on windows to run?